charts/kong/README.md

```diff
@@ -343,13 +461,17 @@ individual services: see values.yaml for their individual default values.
 * `manager`
 * `portal`
 * `portalapi`
+* `cluster`
 * `status`
 
 `status` is intended for internal use within the cluster. Unlike other
 services it cannot be exposed externally, and cannot create a Kubernetes
 service or ingress. It supports the settings under `SVC.http` and `SVC.tls`
 only.
 
+`cluster` is used on hybrid mode control plane nodes. It does not support the
+`SVC.http.*` settings, as cluster communications must be TLS-only.
```

Comment:
Consider using blockquote in markdown to make this stand out:
```
> `cluster` is used ...
```

Not a blocker.

Reply:
No action on this one, leaving as-is based on discussion elsewhere.